dotCMS/src/main/java/com/dotcms/cms/login/LoginServiceAPIImpl.java

```diff
@@ -0,0 +1,537 @@
+package com.dotcms.cms.login;
+
+import static com.dotmarketing.util.CookieUtil.createJsonWebTokenCookie;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import com.dotcms.api.web.HttpServletRequestThreadLocal;
+import com.dotcms.auth.providers.jwt.JsonWebTokenUtils;
+import com.dotcms.cms.login.events.LDAPLoginEventListener;
+import com.dotcms.cms.login.events.LoginEvent;
+import com.dotcms.cms.login.events.LoginEventListener;
+import com.dotcms.cms.login.events.LoginEvents;
+import com.dotcms.enterprise.BaseAuthenticator;
+import com.dotcms.enterprise.LDAPImpl;
+import com.dotcms.enterprise.LicenseUtil;
+import com.dotcms.enterprise.PasswordFactoryProxy;
+import com.dotcms.enterprise.cas.CASAuthUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.enterprise.cas.CASAuthUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)